pkg/api/types.go

```diff
@@ -455,6 +455,9 @@ type PersistentVolumeSpec struct {
 	// means that this volume does not belong to any StorageClass.
 	// +optional
 	StorageClassName string
+	// A comma-separated list of mount options. Not validated--mount will simply fail if one is invalid.
+	// +optional
+	MountOptions *string
```

Comment:
Since beta version was implemented using annotations and that by definition means key and value has to be string (unless we perform special decoding of annotation and since this is exposed to end user, it gets hairy fast) - that was the main reason we gave up on the idea of the mount option being `[]string` . It is probably still not a very good justification. But I think that - using string here is fine as well.